src/core/annotation.js

```diff
@@ -620,7 +620,6 @@ var TextWidgetAnnotation = (function TextWidgetAnnotationClosure() {
     WidgetAnnotation.call(this, params);
 
     this.data.textAlignment = Util.getInheritableProperty(params.dict, 'Q');
-    this.data.hasHtml = !this.data.hasAppearance && !!this.data.fieldValue;
```

Comment:
Fixed. I added it to `WidgetAnnotationElement`. Will move to `TextWidgetAnnotationElement` in #6172